forge16/src/main/java/com/envyful/poke/removal/forge/command/PokeRemovalCommand.java

```diff
@@ -18,7 +31,96 @@ public class PokeRemovalCommand {
 
     @CommandProcessor
     public void onCommand(@Sender ICommandSource sender, String[] args) {
+        if(args.length == 0){
+           Reload(sender);
+           return;
+        }
+
+        if(args[0].contains("reload") || args[0].isEmpty()){
+            Reload(sender);
+            return;
+        }
+
+        if (args[0].contains("specs")) {
```

Comment:
This has no place in this mod. If someone wants to learn about specs they can go to the Pixelmon wiki this is one of many mods that uses specs and as such it would make no sense fo this to be in here

Reply:
That's a good point I will solve it as soon as possible with a new commit.